### DISCUSSION TITLE
Can I define the solve order of the equations?

### DISCUSSION THREAD

**Discussion by Ethan-xj** [idaholab/moose#21654] [Q&A General] [answered]
For example, if I want to firstly solve 3 equations(variables are v t and p) in block1 ang block2. When converged, solve the 4th equation(variable is w), put w into the frist 3 equations and solve the 3 equations again...until converged.
Like the SIMPLE algorithm, can I define this by myself? If I can, where should I write the code?
All I know about MOOSE now is that I write the equations into Kernel and MOOSE will solve the equations at the same time. But I think there must be a way to solve the equations in a user-defined order.

**Comment by GiudGiud** [comment, marked as answer]
Hello

This would be a segregated solver and we do not currently support these. There's ongoing work on this. 

For now you may use MultiApps to accomplish this

Guillaume

**Reply by Ethan-xj** [reply]
OK. This is helpful. Thank you!
